### DISCUSSION TITLE
[question] Versioning And Stability

### DISCUSSION THREAD

**Discussion by jmacvey** [ExpediaGroup/graphql-kotlin#1615] [Q&A] [answered]
Hello, Maintainers!

Your docs show that 7.x.x is latest version and that 6.x.x is no longer actively maintained, but maven is indicating 7.x.x. is still in alpha, and 6.3.x was released after that version.

Seems like these are conflicting statements.  Couple questions result:

(1) Are you all considering your "alpha" builds as stable but in testing per convention on alpha builds, or ready for production use as implied by your docs?
(2) What's the support on 6.x.x?  Is it truly no longer maintained?

Thanks in advance!

**Comment by samuelAndalon** [comment]
Hello,

6.x.x is actively maintained but new features are developed against 7.x.x which reflects main branch, which releases are still on alpha, we should probably modify the warning message on 6.x.x docs.

**Comment by dariuszkuc** [comment, marked as answer]
Hello 👋 

> Your docs show that 7.x.x is latest version and that 6.x.x is no longer actively maintained

Thanks for highlighting that, this is indeed an issue with docusaurus setup. Docs default to the current development branch (which is 7.x release). Per GH releases [6.3.2](https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/6.3.2) is the latest stable release.

> (1) Are you all considering your "alpha" builds as stable but in testing per convention on alpha builds, or ready for production use as implied by your docs?

We follow semantic versioning and use `alpha` to indicate this is a pre-prelease, meaning there could be some additional breaking changes to the API (i.e. updating between `alpha.0` and `alpha.1` may require some additional code changes on your side). We release alpha versions so people can test those out and use new features sooner than the final major release.

> (2) What's the support on 6.x.x? Is it truly no longer maintained?

With the start of the next major version (e.g. v7), all new features by default go the main branch (next major). Depending on the feature and whether it is possible to backport it, they might be also backported to previous major (e.g. v6).

**Reply by jmacvey** [reply]
Thanks for the clarifications!
